[bug] Dependency on `release-it`

I started plating with freshly created __Solid Start__ project that was scaffolded via `yarn create solid` and found that the package `solid-dropzone` cause an error that the devserver can't get `release-it`. The traceback below 
```
  Local:   http://localhost:3000/
  ➜  Network: use --host to expose
  ➜  Inspect: http:/localhost:3000/__inspect/
  ➜  press h to show help
✘ [ERROR] Could not resolve "release-it"

    node_modules/@release-it/keep-a-changelog/index.js:1:27:
      1 │ const { Plugin } = require('release-it');
        ╵                            ~~~~~~~~~~~~

  You can mark the path "release-it" as external to exclude it from the bundle, which will remove
  this error. You can also surround this "require" call with a try/catch block to handle this
  failure at run-time instead of bundle-time.

An unhandled error occured: Error: Build failed with 1 error:
node_modules/@release-it/keep-a-changelog/index.js:1:27: ERROR: Could not resolve "release-it"
    at failureErrorWithLog (/Users/mirus/code/eigurt/node_modules/vite/node_modules/esbuild/lib/main.js:1604:15)
    at /Users/mirus/code/eigurt/node_modules/vite/node_modules/esbuild/lib/main.js:1056:28
    at runOnEndCallbacks (/Users/mirus/code/eigurt/node_modules/vite/node_modules/esbuild/lib/main.js:1476:61)
    at buildResponseToResult (/Users/mirus/code/eigurt/node_modules/vite/node_modules/esbuild/lib/main.js:1054:7)
    at /Users/mirus/code/eigurt/node_modules/vite/node_modules/esbuild/lib/main.js:1166:14
    at responseCallbacks.<computed> (/Users/mirus/code/eigurt/node_modules/vite/node_modules/esbuild/lib/main.js:701:9)
    at handleIncomingPacket (/Users/mirus/code/eigurt/node_modules/vite/node_modules/esbuild/lib/main.js:756:9)
    at Socket.readFromStdout (/Users/mirus/code/eigurt/node_modules/vite/node_modules/esbuild/lib/main.js:677:7)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:324:12)
```

When I added `release-it` as a dev dependency the error was gone.

--- a/README.md
+++ b/README.md
@@ -8,22 +8,48 @@
 
 Drag and drop file input library for SolidJS.
 
-> **Note**: This is a port of [`react-dropzone`](https://github.com/react-dropzone/react-dropzone)
+> **Note**: This is a SolidJS port of [`react-dropzone`](https://github.com/react-dropzone/react-dropzone)
 
 ## Quick start
 
 Install it:
 
 ```bash
-npm i solid-dropzone
+npm i @soorria/solid-dropzone
 # or
-yarn add solid-dropzone
+yarn add @soorria/solid-dropzone
 # or
-pnpm add solid-dropzone
+pnpm add @soorria/solid-dropzone
 ```
 
 Use it:
 
 ```tsx
-import { createDropzone } from 'solid-dropzone'
+import { createDropzone } from '@soorria/solid-dropzone'
+```
+
+## Example
+
+> Adapted from the first example here: https://github.com/react-dropzone/react-dropzone/#usage
+
+```tsx
+import { useDropzone } from '@soorria/solid-dropzone'
+
+function MyDropzone() {
+  const onDrop = (acceptedFiles: File[]) => {
+    // Do something with the files
+  }
+  const dropzone = useDropzone({ onDrop })
+
+  return (
+    <div {...dropzone.getRootProps()}>
+      <input {...dropzone.getInputProps()} />
+      {
+        dropzone.isDragActive ?
+          <p>Drop the files here ...</p> :
+          <p>Drag 'n' drop some files here, or click to select files</p>
+      }
+    </div>
+  )
+}
 ```

--- a/dev/App.tsx
+++ b/dev/App.tsx
@@ -1,10 +1,9 @@
 import { Component, splitProps, For, mergeProps, Show, ComponentProps } from 'solid-js'
 import { createDropzone, CreateDropzoneProps, DropEvent, FileRejection } from '../src'
 
-const Dropzone = (props: Partial<CreateDropzoneProps> & { title: string }) => {
-  const [local, dropzoneProps] = splitProps(props, ['title'])
+const Dropzone = (props: CreateDropzoneProps & { title: string }) => {
   const dropzone = createDropzone(
-    mergeProps(dropzoneProps, {
+    mergeProps(props, {
       onDrop: (files: File[], rejections: FileRejection[], event: DropEvent | null) => {
         props.onDrop?.(files, rejections, event)
       },
@@ -15,7 +14,7 @@ const Dropzone = (props: Partial<CreateDropzoneProps> & { title: string }) => {
 
   return (
     <div class="space-y-6">
-      <h2 class="font-bold text-2xl">{local.title}</h2>
+      <h2 class="font-bold text-2xl">{props.title}</h2>
       <div
         {...dropzone.getRootProps({
           class: 'rounded-xl border-2 border-purple-700 min-h-[8rem] relative overflow-hidden',

--- a/src/index.tsx
+++ b/src/index.tsx
@@ -104,7 +104,7 @@ const getInitialState = () => ({
   acceptedFiles: [] as File[],
   fileRejections: [] as FileRejection[],
 })
-export const createDropzone = (_props: CreateDropzoneProps) => {
+export const createDropzone = (_props: CreateDropzoneProps = {}) => {
   const props = mergeProps(
     {
       disabled: false,

